### PR TITLE
fix(mme): Add libsqlite3 dependency in containers for OAI pipeline

### PR DESCRIPTION
**Pull request by ssanadhya**
Signed-off-by: Shruti Sanadhya <ssanadhya@fb.com>

## Summary

The changes in PR https://github.com/magma/magma/pull/13470 require libsqlite to be present on all dev containers. This PR adds the changes in https://github.com/magma/magma/pull/13470#issuecomment-1209363734 to make it work on Magma-OAI-Jenkins CI pipeline.

## Test Plan

N/A

### CODE REVIEW

**Comment by ssanadhya** [comment]
@rdefosse , can you help me understand why this PR is failing on Magma-OAI-Jenkins pipeline? The [logs](https://jenkins-oai.eurecom.fr/job/MAGMA-MME-production/17571/artifact/build_results_magma_oai_mme.html) only show warnings, no errors.

**Comment by nstng** [comment]
> @rdefosse , can you help me understand why this PR is failing on Magma-OAI-Jenkins pipeline? The [logs](https://jenkins-oai.eurecom.fr/job/MAGMA-MME-production/17571/artifact/build_results_magma_oai_mme.html) only show warnings, no errors.

for me `apt list libsqlite3x` has no candidates on ubuntu 20.04

**Comment by sebathomas** [comment]
Maybe it should be this package in Ubuntu Focal?
https://packages.ubuntu.com/focal/libsqlite3-0

File: ci-scripts/docker/Dockerfile.mme.ci.ubuntu18 [hunk 1]
```diff
@@ -70,6 +70,7 @@ RUN apt-get update && \
       net-tools \
       tshark \
       tzdata \
+      libsqlite3x \
```
Comment (rdefosse) [inline ci-scripts/docker/Dockerfile.mme.ci.ubuntu18]: @ssanadhya 

Sorry. It should be `libsqlite3-0`.

File: ci-scripts/docker/Dockerfile.mme.ci.rhel8 [hunk 2]
```diff
@@ -65,6 +65,7 @@ RUN yum update -y && \
       libubsan \
       libasan \
       liblsan \
+      libsqlite3x \
```
Comment (rdefosse) [inline ci-scripts/docker/Dockerfile.mme.ci.rhel8]: @ssanadhya 

Sorry, it should be just `sqlite`

File: lte/gateway/docker/mme/Dockerfile.rhel8 [hunk 3]
```diff
@@ -45,6 +45,8 @@ RUN rm /etc/rhsm-host && \
     protobuf-compiler \
     ruby \
     ruby-devel \
+    libsqlite3x \
```
Comment (rdefosse) [inline lte/gateway/docker/mme/Dockerfile.rhel8]: Sorry. It should be `sqlite`

File: lte/gateway/docker/mme/Dockerfile.rhel8 [hunk 4]
```diff
@@ -45,6 +45,8 @@ RUN rm /etc/rhsm-host && \
...
+    libsqlite3x-devel \
```
Comment (rdefosse) [inline lte/gateway/docker/mme/Dockerfile.rhel8]: and `sqlite-devel`

File: lte/gateway/docker/mme/Dockerfile.rhel8 [hunk 5]
```diff
@@ -427,6 +429,7 @@ RUN yum update -y && \
       libubsan \
       libasan \
       liblsan \
+      libsqlite3x \
```
Comment (rdefosse) [inline lte/gateway/docker/mme/Dockerfile.rhel8]: And here just `sqlite`

**Review by rdefosse — CHANGES_REQUESTED** [review]

**Comment by ssanadhya** [comment]
Thanks for the review @rdefosse ! I have revised the PR and all required CI checks are green now. Could you please approve and merge this?

**Review by rdefosse — APPROVED** [review]

**Pull request merged**